.env.sample

```diff
@@ -0,0 +1,10 @@
+PARTNER_KEY="secretpassword"
+PARTNER_REGISTER_URL="https://partner.test/api/v1/partners"
+PARTNER_APPROVAL_URL="https://partner.test/api/v1/approve"
+PARTNER_BASE_URL="partner.test"
+FLIPPER_USERNAME="admin"
+FLIPPER_PASSWORD="password"
+SIDEKIQ_USERNAME="admin"
+SIDEKIQ_PASSWORD="password"
+DIAPERBANK_KEY="secretpassword"
```

Comment:
I _think_ that these last two variables are the only ones that partner currently uses no?